templates/default/vlan/vlanlist.html

```diff
@@ -44,8 +44,8 @@ <h1>{$pagetitle}</h1>
             <td>
                 {$v.vlanid}
             </td>
-            <td>
-                {$v.description}
+            <td {tip text="{$v.description}"}>
+                {$v.description|truncate:40:" ...":true}
```

Comment:
Zamiast trzech kropek lepiej użyć wielokropka, który jest przeznaczony do tego (tak jak pisałem wcześniej): `&ellip;`

Reply:
Dopiero obczaiłem różnicę ... jeden symbol zamiast trzech kropek. To w zasadzie w całym LMS można by było poprawić - wszędzie tam gdzie jest truncate.

Reply:
To tutaj niech będzie tak jak w całym LMS a ja puszczę globalną zmianę w całym LMS, żeby wszędzie było jednakowo.

Reply:
OK